packages/now-python/index.js

```diff
@@ -10,7 +10,8 @@ const downloadAndInstallPip = require('./download-and-install-pip');
 async function pipInstall(pipPath, workDir, ...args) {
   console.log(`running "pip install --target ${workDir} ${args.join(' ')}"...`);
   try {
-    await execa(pipPath, ['install', '--target', '--upgrade', '.', ...args], {
+    await execa(pipPath, ['install', '--target', '.', ...args], {
```

Comment:
```suggestion
    await execa(pipPath, ['install', '--target', '--upgrade', '.', ...args], {
```

I think we need to keep `--upgrade`. It was just added by @NathanielHill in #294

Reply:
@styfle I actually removed it due to this error.
![image](https://user-images.githubusercontent.com/5785105/55445161-1b463080-55e4-11e9-94ac-749673610176.png)

Reply:
```suggestion
    await execa(pipPath, ['install', '--target', '.', '--upgrade', ...args], {
```
This should be the right order for the flag. Note that `--target` is not a key-only flag.

Reply:
Thank you, it's working! 🙌